CMakeLists.txt

```diff
@@ -151,12 +154,28 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/examples/CMakeLists.txt")
 endif()
 
 
-install(TARGETS ${XMP_INSTALLS}
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-        INCLUDES DESTINATION "include")
+include(GNUInstallDirs)
```

Comment:
Please put includes into the top of the file to make an order

Reply:
I put it where it's needed, but OK will do.

Question: Do we really need `install`ing for MSVC builds?

Reply:
Yes, needed, the direct example of use:
- Taking this library and others as ExternalProject
- Making the dedicated hive of libraries and set it as "CMAKE_INSTALL_PREFIX"
- All built libraries will appear in that hive of libraries
- The main project will use these libraries directly

Reply:
As an example, my TheXTech is built also for Windows ARM64, uses MSVC, and it does the same as other platforms and toolchains: all libraries were installed into sub-directory of the build directory that got set as "CMAKE_INSTALL_PREFIX", and all these libraries getting easily used by the main project itself.

Reply:
Also, the `install()` is useful while crafting various packages, and there is the `cpack` tool that allows you quickly make the binary package of the library if you specify all flags required for certain package type, and the `install()` will be needed

Reply:
> Yes, needed, [...]

OK

Reply:
> I put it where it's needed, but OK will do.

Well, looking at it more, I think it is good where it is